### PR TITLE
[DomCrawler] Rename UriExpander.php -> UriResolver

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | refs https://github.com/symfony/symfony-docs/pull/13054#discussion_r377138605
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/13054 // don't open a new issue in doc repo

### CODE REVIEW

File: src/Symfony/Component/DomCrawler/UriResolver.php [hunk 1]
```diff
@@ -12,22 +12,23 @@
...
  *
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
-class UriExpander
+class UriResolver
 {
     /**
-     * Expand an URI according to a current Uri.
+     * Resolve an URI according to a current Uri.
```
Comment (nicolas-grekas) [inline src/Symfony/Component/DomCrawler/UriResolver.php]: let's use "base URI" instead of "current URI" everywhere also, to match the RFC?

Comment (lyrixx) [reply src/Symfony/Component/DomCrawler/UriResolver.php]: :+1: good idea.

File: src/Symfony/Component/DomCrawler/CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ CHANGELOG
 -----
 
 * Added an internal cache layer on top of the CssSelectorConverter
-* Added `UriExpander` to expand an URL according to another URL
+* Added `UriResolved` to resolve an URI according to a base URI
```
Comment (ro0NL) [inline src/Symfony/Component/DomCrawler/CHANGELOG.md]: `UriResolver` :)

File: src/Symfony/Component/DomCrawler/UriResolver.php [hunk 3]
```diff
@@ -12,22 +12,23 @@
...
  *
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
-class UriExpander
+class UriResolver
 {
     /**
-     * Expand an URI according to a current Uri.
+     * Resolve an URI according to a base URI.
```
Comment (OskarStark) [inline src/Symfony/Component/DomCrawler/UriResolver.php]: ResolveS

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**